### ISSUE TITLE
[Bug.Minor] Collecting infocaches causes a "Destroyed unit pointer" to spawn

### ISSUE THREAD

**Issue by Regallion** [KnuckleCracker/CW4-bug-tracker#496]
1. Create an info caches with an ADA-type message. Make sure to have an actual message specified.
2. When cache is collected, you will see the purple 'Where was a unit destroyed" pointer pointing at the cache for 1 frame.
3. I cannot screenshot this even with frame advance but it's occasionally notable.
 Can replicate easily and at-will.

**Comment by knucracker** [comment]
The pointer should be present and visible while the ada log message is open. In theory you can still pan the map and see what the pointer is pointing at (the infocache that was collected).

**Comment by Regallion** [comment]
That is true, although it never occurred to me to pan the map around while viewing an ADA message. (That would be because of the background overlay rendering it hard to see).

But if that is intended, why is it not true for GameMessage caches?
